help/destinations/catalog/adobe/marketo-engage-person-sync.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Marketo Engage Person Sync
+description: Use the Marketo Engage Person Sync connector to stream updates from a person audience to the corresponding records in your Marketo Engage.
```

Comment:
let's add this to the metadata:
badgeBeta: label="Beta" type="Informative"

this will display a beta badge on the page